hypershift-operator/controllers/nodepool/nodepool_controller.go

```diff
@@ -1246,6 +1395,13 @@ func defaultAndValidateConfigManifest(manifest []byte) ([]byte, error) {
 	case *v1alpha1.ImageContentSourcePolicy:
 	case *mcfgv1.KubeletConfig:
 	case *mcfgv1.ContainerRuntimeConfig:
+	case *tunedv1.Tuned:
```

Comment:
this effectively enables tunedv1.Tuned as valid to be passed through .config knob which is not what we want. We need a different function to validate this.

Reply:
Good point. I removed this and added a separate function specifically for the Tuned manifests.